### ISSUE TITLE
Dropdown default is "none". Need a way to make it non-specified / all / any.

### ISSUE THREAD

**Issue by LucySparks** [pdaleramirez/super-filter#14]
We've set your Super Filter plugin up to use an "and" relationship between two category fields.

All our items have values for categories 1 and 2 set. 

Our category 1 renders as a checkbox list, and category 2 as a dropdown. 

"None" is the default value for the dropdowns, it lists the actual category values beneath "none". 

The issue is: if you're looking for something specific in category 1, but you don't mind / set which value you want for category 2, the search will always return nothing, because the dropdown is specifically requesting a _none_ or _null_ value for category 2 but all out items have values for category 2. 

Basically, we'd like to know how to change the default "none" value in the dropdown to an "all" value or an "any".

<img width="1180" alt="Screen Shot 2020-10-29 at 9 48 44 am" src="https://user-images.githubusercontent.com/37562179/97506904-f6868a80-19cb-11eb-856d-9d9ef4d2d199.png">

Cheers

**Comment by pdaleramirez** [comment]
@LucySparks It is possible to modify the text "None" by using the template override feature. Instruction here https://github.com/pdaleramirez/super-filter#using-super-filter. You need to input an override folder on search setup page. Then you can copy the example templates in the plugin `pdaleramirez/superfilter/plain/fields/dopdown.html` to your specified override folder. Then on your site templates you can modify the look of your html.

**Comment by LucySparks** [comment]
Hi @pdaleramirez, thanks for your response.
I am already using an override folder structure that has duplicated and adapted the templates from the 'plain' variation in your plugin.
Yes, I can change the field label from `none` to `all` but I'm unclear on what value to pass or assign the dropdown option to return `any` or `all` options for category 2, in the filter query. 
At a guess `*` didn't work. 
Any light you could shed would be greatly appreciated. 
Thanks

**Comment by pdaleramirez** [comment]
@LucySparks I see what you meant now, I will make an enhancement and make a release soon.

**Comment by pdaleramirez** [comment]
@LucySparks I've released `1.2.1` a feature that allows you to change the category operator. On the Super filter settings choose "Or" under "Category Relationship Operator" field and save the setting. You can now use the "*" category value on your search parameter that value will return all elements that has category.

**Comment by LucySparks** [comment]
Hi @pdaleramirez 
Thanks for that, that's fixed it!
